examples/Go2Target.ipynb

```diff
@@ -0,0 +1,388 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Figure out what is the rotPA that sets nasmith rotator close to zero.\n",
+    "time_data = await atptg.tel_timeAndDate.next(flush=True, timeout=2)\n",
+    "curr_time_atptg = Time(time_data.tai, format=\"mjd\", scale=\"tai\")\n",
+    "print(curr_time_atptg)\n",
+    "coord_frame_altaz = AltAz(location=location, obstime=curr_time_atptg)\n",
+    "alt_az = radec.transform_to(coord_frame_altaz)\n",
+    "\n",
+    "print(\"slew...\")\n",
+    "# await atmcs.cmd_startTracking.start(timeout=10)\n",
+    "await atptg.cmd_raDecTarget.set_start(\n",
+    "    targetName=target_name,\n",
+    "    targetInstance=ATPtg.TargetInstances.CURRENT,\n",
+    "    frame=ATPtg.CoordFrame.ICRS,\n",
+    "    epoch=2000,  # should be ignored: no parallax or proper motion\n",
+    "    equinox=2000,  # should be ignored for ICRS\n",
+    "    ra=radec.ra.hour,\n",
+    "    declination=radec.dec.deg,\n",
+    "    parallax=0,\n",
+    "    pmRA=0,\n",
+    "    pmDec=0,\n",
+    "    rv=0,\n",
+    "    dRA=0,\n",
+    "    dDec=0,\n",
+    "    rotPA=180.-alt_az.alt.deg,\n",
+    "    rotFrame=ATPtg.RotFrame.FIXED,\n",
+    "    rotMode=ATPtg.RotMode.FIELD,\n",
+    "    timeout=10\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In case you need to stop tracking, use the next cell!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await atptg.cmd_stopTracking.start(timeout=10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Use the next cell in case you need to offset to center the target on the FoV.\n",
+    "\n",
+    "This will set total offsets. So, if you say `el=0` and `az=-30` and then later you do `el=30` and `az=0.` , it will reset the offset in azimuth to zero and make an offset of 30arcs in elevation. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await atptg.cmd_offsetAzEl.set_start(el=0., \n",
+    "                                     az=-100., \n",
+    "                                         num=0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to make persistet offsets you can use the following method. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await atptg.cmd_offsetAzEl.set_start(el=0., \n",
+    "                                     az=-100., \n",
+    "                                     num=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to add your offset to a pointing model file, do the following. "
```

Comment:
Is this really part of a standard slew? It feels outside the scope of a slew example. Might belong in a "create a pointing model" example?